README.md

```diff
@@ -114,3 +159,94 @@ From your projects folder:
 Now your web's chain-adapters have a symlink to your lib's
 
 You can use `yarn run show-linked-packages` to show what is currently symlinked via yarn
+
+## Developer Onboarding
+
+1.  Create your own fork of the project.
+2.  (optional) Install [Pull](https://github.com/apps/pull) so that your fork automatically stays up-to-date with changes from the ShapeShift repo.
+3.  Install [Git Credential Manager](https://github.com/GitCredentialManager/git-credential-manager/releases)
+    so that you don't need to authenticate when executing git commands.
+
+        (Note that the Linux packages for versions 2.0.567 and 2.0.605, which are the two latest versions at time of writing, are broken. Until a fixed version is released, you'll want to use [2.0.498](https://github.com/GitCredentialManager/git-credential-manager/releases/tag/v2.0.498).)
+
+        On Debian-based Linux distros, including Ubuntu, Mint, and Pop OS, download the `.deb` package for the version you want to install. Then run:
+
+        ```sh
+        sudo dpkg -i '<path to .deb file>'
+        git-credential-manager-core configure
+        git config --global credential.credentialStore secretservice
+        ```
```

Comment:
i think this is general developer skills/preferences and not specific to this repo, and should be excluded

Reply:
I'd support factoring this out into the docs repo but IMO that's better considered as a separate task